app/views/shared/_head.html.haml

```diff
@@ -1,6 +1,7 @@
 %meta{ "http-equiv" => "content-type", "content" => "text/html; charset=UTF-8" }
 - # use gsub to strip out html tags in the title to prevent html from showing #62643
 = render partial: "/shared/title", locals: { title: yield(:h1).gsub(/<[^>]*>/, " ") }
+%meta{:name => "viewport", :content => "width=device-width; initial-scale=1.0;"}
```

Comment:
We like 1.9 hash syntax (there are still some residual rockets from the 1.8 days, but we're trying to move away from them).